Packages/Store/Sources/Requests/ContactAddressListRequest.swift

```diff
@@ -0,0 +1,50 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+
+import Foundation
+import GRDBQuery
+import GRDB
+import Primitives
+import Combine
+
+public struct ContactAddressListRequest: ValueObservationQueryable {
+    public static var defaultValue: [ContactAddress] { [] }
+    
+    private let contactId: ContactId?
+    private let chain: Chain?
+    private let searchQuery: String?
+    
+    public init(
+        contactId: ContactId?,
```

Comment:
```suggestion
        contactId: ContactId?,
```
should not be optional